ztp/policygenerator/utils/filesHandler.go

```diff
@@ -55,24 +55,35 @@ func (fHandler *FilesHandler) ReadSourceFile(fileName string) ([]byte, error) {
 }
 
 func (fHandler *FilesHandler) ReadSourceCRFile(fileName string) ([]byte, error) {
-	var dir = ""
-	var err error = nil
-	var ret []byte
+	var (
+		gitDir   = ""
+		localDir = ""
+		err      error
+		fileByte []byte
+	)
 
-	ex, err := os.Executable()
+	// current working directory in git
+	gitDir, err = os.Getwd()
 	if err != nil {
 		return nil, err
 	}
-	dir = filepath.Dir(ex)
-	ret, err = fHandler.ReadFile(dir + "/" + SourceCRsPath + "/" + fileName)
+	sourceCRPathGit := gitDir + "/" + SourceCRsPath + "/" + fileName
 
-	// added fail safe for test runs as `os.Executable()` will fail for tests
+	// path of the local executable
+	localDir, err = os.Executable()
```

Comment:
I believe we should read first the `os.ReadFile(sourceCRPathGit)`  if it fail to read the file then read local `os.Executable()`

Reply:
This is exactly what we are doing at line 80 below.